### PR TITLE
Added #88 Feature: Create a new store for new projects 

### PR DESCRIPTION
**Pull request by Rahul139-stack**
please review and approve it

### CODE REVIEW

File: src/store/Modules/NewProject.ts [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+import store from "../index"
+const storeValue = localStorage.getItem("vuex")
+const newProjectInStore = storeValue ? JSON.parse(storeValue).NewProject : false
+@Module({
+  name: "NewProject",
+  store: store,
+  dynamic: true,
+  preserveState: newProjectInStore,
+})
+export default class NewProject extends VuexModule {
```
Comment (wildone) [inline src/store/Modules/NewProject.ts:11]: this class shoule be called Projects. 
this file should be called Projects.

**Review by wildone — CHANGES_REQUESTED** [review]

File: src/store/Modules/Projects.ts [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+const storeValue = localStorage.getItem("vuex")
+const projectsInStore = storeValue ? JSON.parse(storeValue).Projects : false
+@Module({
+  name: "Projects",
+  store: store,
+  dynamic: true,
+  preserveState: projectsInStore,
+})
+export default class Projects extends VuexModule {
+  projects = {
```
Comment (wildone) [inline src/store/Modules/Projects.ts:12]: lets remove this root element.

**Review by wildone — CHANGES_REQUESTED** [review]
need to fix conflict.

**Comment by wildone** [comment]
looking good!

**Review by ankitvats1 — APPROVED** [review]

File: src/store/Modules/NewProject.ts (hunk 1 above)
Comment (Rahul139-stack) [reply src/store/Modules/NewProject.ts]: Class name and file name modified

File: src/store/Modules/Projects.ts (hunk 2 above)
Comment (Rahul139-stack) [reply src/store/Modules/Projects.ts]: Root element removed

**Review by ankitvats1 — APPROVED** [review]

**Review by ankitvats1 — APPROVED** [review]

**Review by wildone — APPROVED** [review]

**Pull request merged**